.travis.yml

```diff
@@ -1,63 +0,0 @@
-sudo: required
-dist: trusty
-language: erlang
-otp_release:
-  - 20.2.3
-before_install:
-  - ./ci before_install "${JOB:?}" "${TRAVIS_BUILD_DIR:?}"
-install:
-  - ./ci install "${JOB:?}" "${TRAVIS_BUILD_DIR:?}"
-script:
-  - ./ci script "${JOB:?}" "${TRAVIS_BUILD_DIR:?}"
-after_failure:
-  - ./ci after_failure "${JOB:?}" "${TRAVIS_BUILD_DIR:?}"
-deploy:
-  - provider: releases
-    api_key:
-      # See https://docs.travis-ci.com/user/deployment/releases/#Authenticating-with-an-OAuth-token
-      secure: SsNvB5u94rozdGDwn4jhkbR9RoN/KQQYij1vXiJzmk1y997GsLG/skBlgqmpuGQap+RgvDHWAJ0sxZh0H+xHZB78SKmhryvLdPTmuN4B3qgvIuJPDtvrogtEDvZ8SXQXOXPlH/o7hBvyfebUXDIqn8UU8DUaexnpmwgxjI+AezHIFRtwyd29pDBQHLebHo3OoB6hsKM6mqxb/SfDXhktSfoqlZ2zQvU1boXzwMo/S54OMY6z2Ui0xbdXilkxwki8yW88ouHsyekmErYMGXS/j1gubxDkMQ+ZO+BcbFR4ZRt2a8FUkqr2pymzf4lBACkxSzzSKJrC8Y49ENpqsCHiZE1ATNsoNKMn8XNynf/N4H6lc8np3BOdcBA+lGz40CAZI8kMal+pGXKsex7lbbvs/gkNufD9LCiW4JAkA/ndFPYobuxE07nraSvD/b8sP2H5uSMXJWMwZMgudmUZh2kdwZFCFTyIAbk8AC94wpJFII/45rL9yXf4tfbBqTUgPnGs0cUuKCRPkdQ5uNnHNzSpi/CQhHob3JVXBJ1Ve67SiUDU44kvkv6PdnX1i95wJkcPeX485amW0w2qX3CwGrwm9pnoOZTRkfBXMRitfxQ2i42r6T7zZTgKjPHDg8Stf6ZPqbmTLj4++hni8bGNICs5caZKinKCH3e60PL5JzPZ0u8=
```

Comment:
After this PR is merged we shall delete any traces of this token before testing deployment to UAT.